Firmware/Marlin_main.cpp

```diff
@@ -4831,18 +4822,18 @@ if(eSoundMode!=e_SOUND_MODE_SILENT)
 	#### Parameters
       - `N` - Number of mesh points on x axis. Default is 3. Valid values are 3 and 7.
       - `R` - Probe retries. Default 3 max. 10
-      - `V` - Verbosity level 1=low, 10=mid, 20=high. It can be only used if firmware has been compiled with SUPPORT_VERBOSITY active.
+      - `V` - Verbosity level 1=low, 10=mid, 20=high. It only can be used if the firmware has been compiled with SUPPORT_VERBOSITY active.
       
       Using the following parameters enables additional "manual" bed leveling correction. Valid values are -100 microns to 100 microns.
-	  #### Additional Parameters
+    #### Additional Parameters
```

Comment:
Not sure that this will break doxygen ... you have to follow very strickt in the format.
Did you run doxygen on your branch?

Reply:
Yes. I didn’t see problems when I changes it. I’ll check again if it’s needed

Reply:
The G80 area looks fine to me. Please also check it on your part, but I think it will be the same